doc/nrf/protocols/matter/getting_started/hw_requirements.rst

```diff
@@ -79,6 +79,10 @@ Values are provided in kilobytes (KB).
       +------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
       | :ref:`Window Covering <matter_window_covering_sample>` (Release) |            28 |               688 |              4 |         32 |         752 |                             213 |
       +------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Thermostat <matter_thermostat_sample>` (Debug)             |            ?? |               ??? |              ? |         ?? |         ??? |                             ??? |
+      +------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Thermostat <matter_window_covering_sample>` (Release)      |            ?? |               ??? |              ? |         ?? |         ??? |                             ??? |
+      +------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
```

Comment:
@doublemis1 , @jnsgsr , can one of you help with updating the memory values script for the new sample?

Reply:
we need to add them to test script to automatically get the values
I will try to do it manually to fill needed values